keepassxc_browser/cli.py

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/python3
+
+import sys
+import os
+import re
+import subprocess
+import argparse
+import json
+import tldextract
```

Comment:
missing package dep in https://github.com/hrehfeld/python-keepassxc-browser/blob/610ea634c2832d6c3948a6851adbc59917bf0119/setup.py#L7